impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaKafkaNode.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.testing.kafka.invm;
+
+import java.io.PrintStream;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.kafka.common.utils.Time;
+import org.jetbrains.annotations.NotNull;
```

Comment:
nit: we should use the findbugs annotations.

 ```suggestion
import edu.umd.cs.findbugs.annotations.NonNull;
```

Reply:
:( need to set up IDEA properly, I keep getting these inserted automatically on method extract.

Reply:
Here's one I made earlier https://github.com/kroxylicious/kroxylicious/pull/632/commits/c771ce542099019984def3f9ac8fc7b6125e42bd